integration_tests/dbt_project.yml

```diff
@@ -46,6 +46,8 @@ vars:
     # snowplow__atomic_schema: "{{target.schema ~ '_snplw_web_int_tests'}}"
     # snowplow__events_table: 'snowplow_web_events_stg'
     snowplow__events: "{{ ref('snowplow_web_events_stg') }}"
+    snowplow__atomic_schema: "{{ target.schema }}_snplw_web_int_tests"
```

Comment:
Can you delete line 46 in this case? I needed it for the base macro stuff but it's the same as you've done now.